squeemishly/dnd_designer
# Create character backend

### What does this PR do?
- create the character on the backend, including: create character, create user_character, and create character_class
- change rendering of races to pull from API call instead of hard coding race names
- remove the subrace dropdown menu for races without subraces

### What do you need to do to set up your computer?
- knex migrate:latest
- knex seed:run

### How do I test it?
Try making a character. Feel free to add a console.log in Character.createCharacter in both sides of the conditional to see the results of the our labor.
